.hopper/config.yml

```diff
@@ -1,19 +1,19 @@
 version: '1'
 
 services:
-  internal-web:
+  web:
     containerDefinitions:
-      internal-web:
+      web:
         cpu: 1024
         memory: 1024
         essential: true
         command: "bundle exec dotenv puma -I . -p 8000 -C config/puma.rb"
         portMappings:
```

Comment:
btw, I think you no longer need to add this. Hopper already knows about the port and injects this into the task definition

Reply:
I _think_ it's needed because we had explicitly mentioned a port in app-infrastructure. If you don't strongly feel about removing this, I'd prefer to keep things as they are and reduce the time I'd potentially spend on experiment with this to get it to work.

Reply:
It used to be the case, until @marcinwyszynski made a change in hopper to autopopulate port mappings (https://github.com/deliveroo/hopper/pull/234)

It's up to you though, if you have it here, it overrides what Hopper sets. Since this is not something that can be easily changed as it requires the target group to be updated via Terraform. The idea was that since people would need to make a code change in deliveroo/app-infrastructure, let's pass this value from roo_service_... to hopper_service and Hopper can let ECS know what port is the service running.

Reply:
Thanks for explaining! :+1: 
Keeping it as is for now. Can play with this once live.